jenkins-integration/beaker/install/foss/40_add_puppet_bin_to_path.rb

```diff
@@ -0,0 +1,6 @@
+test_name "Add puppet binaries to the $PATH"
+
+hosts.each do |host|
+  on(host, 'echo "export PATH=\$PATH:/opt/puppetlabs/puppet/bin" >> ~/.bashrc')
+  on(host, 'echo "export PATH=\$PATH:/opt/puppetlabs/server/bin" >> ~/.bashrc')
```

Comment:
Do we symlink all the binaries into `/opt/puppetlabs/bin/`?  If so we could just add that path